clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AnnotationsBasedAtomicOperationsRegistry.groovy

```diff
@@ -57,27 +58,29 @@ class AnnotationsBasedAtomicOperationsRegistry extends ApplicationContextAtomicO
       value instanceof AtomicOperationConverter
     }.values().toList()
 
+    converters = converters.findAll { AtomicOperationConverter cv -> cv.acceptsVersion(version) }
```

Comment:
Oh good idea, I know that in the future if we start to lean on this feature we'll want to define how to resolve a match when more than one version can be used (e.g. v1 can be used in place of v1beta1 safely). Putting that in a helper now will make adding that later easier.